internal/server/data/migrator/migrator.go

```diff
@@ -50,6 +54,11 @@ var DefaultOptions = Options{
 
 // New returns a new Migrator.
 func New(db *gorm.DB, options Options, migrations []*Migration) *Migrator {
+	if options.LoadKey == nil {
```

Comment:
Are we going to make it possible to call specific migrations? Otherwise this will be required most of the time I imagine.

Reply:
Specific migrations, no, but it's pretty easy to pass in an arbitrary list of migrations. In the test I plan on running `[0:n]` to simulate applying each migration in order.

In our tests though we don't always pass a `LoadKey`, so I think it has to be optional unless we start using a consistent data key with all of our tests. Should be easy to make it required later if we need to.